18F/federal-grant-reporting
# Display individual findings and initial findings list

TODO:
- [x] Add Findings model managers
- [x] Add "finding" individual detail view
- [x] Add "findings" list view
- [x] Add URLs
- [x] Add initial template files

Steps to review:
1. Create a finding through the admin interface (`/admin`), if you haven't already done so.
2. Go to `finding/1`. You should see an individual finding. (It's just the title for now.)
3. Go to `findings/`. You should see a list of however many findings you've created.
